### ISSUE TITLE
unable to see cursor in search

### ISSUE THREAD

**Issue by rishabhdeepsingh** [morhetz/gruvbox#340]
I am not able to see which line I am at when I search for a keyword. The highlighted text looks the same as searched text.

**Comment by gustaphe** [comment]
I agree that this is a problem. I have a line in `~/.vim/after/.vimrc`:
```vim
hi clear Search  
hi Search cterm=underline ctermfg=3  
```

**Comment by rishabhdeepsingh** [comment]
I am using https://github.com/PeterRincker/vim-searchlight and its really cool.
